app/src/androidTest/java/mozilla/lockbox/screenshots/ScreenshotsTest.kt

```diff
@@ -42,15 +42,15 @@ open class ScreenshotsTest {
     val localeTestRule = LocaleTestRule()
 
     @Test
-    fun testCompleteOnboarding() {
+    fun testThroughoutAllApp() {
 
         Screengrab.setDefaultScreenshotStrategy(UiAutomatorScreenshotStrategy())
-        onView(withId(R.id.buttonGetStarted))
+        onView(withId(R.id.buttonGetStartedManually))
```

Comment:
Most of these can be addressed using UI test robots. 

e.g. see `WelcomeRobot`

```kotlin
welcome {
    tapGetStarted()
}
```